### PR TITLE
Fix build on OpenBSD i386

### PR DESCRIPTION
**Pull request by rhubner**

### CODE REVIEW

File: java/src/test/java/org/rocksdb/PerfContextTest.java [hunk 1]
```diff
@@ -80,6 +81,8 @@ public void testAllGetters() throws RocksDBException, IntrospectionException,
 
   @Test
   public void testGetBlockReadCpuTime() throws RocksDBException {
+    Assume.assumeFalse("We are not running this test on OpenBSD or Windows",
```
Comment (adamretter) [inline java/src/test/java/org/rocksdb/PerfContextTest.java:84]: Why is this test now prohibited on Windows? Presumably it was passing Windows CI before?
If this is memory limited, perhaps again we should change the assumption to `!Environment.is64Bit()`?

**Review by adamretter — COMMENTED** [review]
Just some questions about Assumptions for tests, otherwise LGTM.

File: java/src/test/java/org/rocksdb/PerfContextTest.java (hunk 1 above)
Comment (rhubner) [reply java/src/test/java/org/rocksdb/PerfContextTest.java:84]: I found it when I was developing this issues. We never run it on Windows. There are some test which we don't run via CMake.

**Review by adamretter — APPROVED** [review]
LGTM now thanks.

File: java/src/test/java/org/rocksdb/PerfContextTest.java (hunk 1 above)
Comment (adamretter) [reply java/src/test/java/org/rocksdb/PerfContextTest.java:84]: Okay thanks @rhubner, as agreed we will address the Windows CMake tests issue in a separate PR.

**Review by ajkr — APPROVED** [review]

**Pull request closed without merging**